Python/libraries/recognizers-date-time/recognizers_date_time/date_time/spanish/duration_extractor_config.py

```diff
@@ -103,10 +95,6 @@ def __init__(self):
             SpanishDateTime.SuffixAndRegex)
         self._relative_duration_unit_regex: Pattern = RegExpUtility.get_safe_reg_exp(
             SpanishDateTime.RelativeDurationUnitRegex)
-        self._more_than_regex: Pattern = RegExpUtility.get_safe_reg_exp(
-            SpanishDateTime.MoreThanRegex)
```

Comment:
Why this regex is being erased?

Reply:
It was being defined twice, which might create conflicts and was not required